pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_creating-a-product.adoc

```diff
@@ -0,0 +1,28 @@
+[id="creating-a-product_{context}"]
+= Creating a product
+
+A product must exist in {ProductName} before you can assign a product name to the metadata of a module or assembly. If the product doesn't already exist, you need to create one.
+
+.Prerequisites
+
+* You are logged in.
+* You have Publisher permissions.
+
+.Procedure
+
+. Navigate to menu:Products[New Product].
+
+. Enter the following information:
+  Product Name:: This is the text that appears in {ProductShortName}, and the document preview, and in the user-facing content.
```

Comment:
`The text that appears in {ProductShortName}, the document preview, and in user-facing content.`